packages/components/nodes/documentloaders/Folder/Folder.ts

```diff
@@ -46,6 +46,22 @@ class Folder_DocumentLoaders implements INode {
                 type: 'TextSplitter',
                 optional: true
             },
+            {
+                label: 'Pdf Usage',
+                name: 'pdfUsage',
+                type: 'options',
+                options: [
+                    {
+                        label: 'One document per page',
+                        name: 'perPage'
+                    },
+                    {
+                        label: 'One document per file',
+                        name: 'perFile'
+                    }
+                ],
+                default: 'perPage'
```

Comment:
can we add `additionalParams: true`?

Reply:
also we'll have to increment the version at the top whenever we add a new parameter